php/libraries/NDB_Client.class.inc

```diff
@@ -132,8 +132,8 @@ class NDB_Client
             . $config_additions
         );
         // start php session
-        $sessionOptions = array('cookie_httponly' => true);
-        $sessionOptions['cookie_samesite'] = true;
+        $sessionOptions = ['cookie_httponly' => true];
```

Comment:
The bracket style was changed here to avoid conflicts with main (main is using `[`) when pulling this forward